packages/reassure-measure/src/measure-function.tsx

```diff
@@ -0,0 +1,38 @@
+import { config } from './config';
+import { RunResult, MeasureResults, processRunResults } from './measure-helpers';
+import { showFlagsOuputIfNeeded, writeTestStats } from './output';
+
+interface MeasureFunctionOptions {
+  runs?: number;
+  dropWorst?: number;
```

Comment:
rename to `warmupRuns` to match existing API

Reply:
done it :)